lib/passkit.rb

```diff
@@ -20,18 +20,35 @@ def self.configure
     yield(configuration) if block_given?
   end
 
+  def self.configured?
+    self.configuration&.configured?
+  end
```

Comment:
I've thrown that in here since we use it in our app to conditionally display passkit-related view code only if the gem is properly configured.